README.md

```diff
@@ -121,6 +121,10 @@ ex)
 ```php
 use Cake\Event\Event;
 use Cake\Event\EventListenerInterface;
+use Cake\Http\ServerRequestFactory;
+use Sentry\State\Scope;
+
+use function Sentry\configureScope as sentryConfigureScope;
```

Comment:
In L141, `ServerRequest` is referenced. It also needs to be added.

```php
use Cake\Event\Event;
use Cake\Event\EventListenerInterface;
use Cake\Http\ServerRequest;
use Cake\Http\ServerRequestFactory;
use Sentry\State\Scope;
use function Sentry\configureScope as sentryConfigureScope;
```

cf: https://github.com/Connehito/cake-sentry/blob/master/tests/test_app/app/src/Error/Event/SentryErrorContext.php